docker.Makefile

```diff
@@ -8,10 +8,10 @@ $(warning WARNING: No docker user found using results from whoami)
 DOCKER_ORG                = $(shell whoami)
 endif
 
-CUDA_VERSION              = 12.1.1
+# CUDA_VERSION              = 12.1.1
```

Comment:
Any particular reason for commenting this out?

Changing this to the following will make it so that can actually change it through an env variable. 
```suggestion
CUDA_VERSION             ?= 12.1.1
```